stdlib/private/StdlibCollectionUnittest/LoggingWrappers.swift.gyb

```diff
@@ -167,21 +167,25 @@ public struct LoggingRangeReplaceableCollection<
     base.insert(newElement, at: i)
   }
 
+  @discardableResult
   public mutating func remove(at index: Base.Index) -> Base.Iterator.Element {
     Log.removeAt[selfType] += 1
     return base.remove(at: index)
   }
 
+  @discardableResult
   public mutating func _customRemoveLast() -> Base.Iterator.Element? {
     Log._customRemoveLast[selfType] += 1
     return base._customRemoveLast()
   }
 
+  @discardableResult
   public mutating func _customRemoveLast(n: Int) -> Bool {
```

Comment:
Please don't add the attribute to `_custom*` entry points.  They are special customization entry points, only to be used as implementation for non-`_custom` ones.  Their result should be used.